### ISSUE TITLE
Would love a 1.3.4 release

### ISSUE THREAD

**Issue by HeyBillFinn** [rholder/retrying#62]
I'd like to download from PyPI and be able to use the `after_attempts` decorator kwarg.

Let me know if there's anything I can do to help!

Thanks for creating this project. It's exactly what I need.

**Comment by bersace** [comment]
I agree. Releasing frequenty is good for quality and developer happyness :-)

**Comment by bbayles** [comment]
`retrying` is great, and I'd like to see a new release as well. I'm happy to volunteer to be a co-maintainer, if @rholder is constrained?

**Comment by HeyBillFinn** [comment]
@rholder - I'm also happy to volunteer as a co-maintainer. Anything we can do to help move to a 1.3.4 release?

**Comment by bersace** [comment]
Do we need to fork ?

**Comment by lucagiovagnoli** [comment]
Hi @rholder! Could you please push 1.3.4 to pypi ? In case it's needed, I also can volunteer as co-maintainer.

**Comment by bersace** [comment]
@lucagiovagnoli did you read #65 ? :)

**Comment by lucagiovagnoli** [comment]
@bersace just after posting here. Thanks a lot for the pointer, I'll see if we can switch to tenacity :)

**Comment by bersace** [comment]
@lucagiovagnoli it worth the migration :-)
